spec/models/ability_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe Ability do
+
```

Comment:
You know I believe that even bad standard is better than no standard :smile:, but OK I will not report issues similar to the one above in the feature.